### PR TITLE
fix: Added a check to skip prepareSkinResources for dataImport tasks

### PR DESCRIPTION
**Pull request by jonathanmtran**
##### Checklist

-   [x] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]

##### Description of change

This is an attempt at #213 

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/.github/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/.github/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

File: overlays/uPortal/build.gradle [hunk 1]
```diff
@@ -97,7 +97,21 @@ task skinGenerate() {
 }
 
 // Set up resources from custom skins so we can compile with NodeJS & lessc
-if (skinsDir.exists()) {
+boolean taskIsDataImport = false;
+
+project.gradle.startParameter.taskNames.each { task ->
```
Comment (ChristianMurphy) [inline overlays/uPortal/build.gradle]: Groovy has an `any` helper to see if some element in a collection matches a predicate.
http://groovy-lang.org/groovy-dev-kit.html#_manipulating_lists

Comment (jonathanmtran) [reply overlays/uPortal/build.gradle]: `any` works if one were to execute `./gradlew dataImport ...`, but it wouldn't work for `./gradlew overlays:uPortal:dataImport`

Comment (ChristianMurphy) [reply overlays/uPortal/build.gradle]: Even with

```groovy
project.gradle.startParameter.taskNames.any { it.contains('dataImport') }
```

:question:

Comment (jonathanmtran) [reply overlays/uPortal/build.gradle]: I sit corrected

**Comment by jgribonvald** [comment]
I'm agree it's a good improvment !

**Pull request merged**